public_suffix_list.dat

```diff
@@ -11693,4 +11695,18 @@ yolasite.com
 za.net
 za.org
 
+// Cloud DNS Ltd : http://www.cloudns.net
+// Submitted by Aleksander Hristov <noc@cloudns.net>
+cloudns.cc
+cloudns.asia
```

Comment:
As per the guidelines, you should sort all of these by the right-most label and then work leftward. That is, .asia, .biz, .club (sorting by TLD, then 2LD, then 3LD)